README.md

```diff
@@ -17,6 +17,8 @@ The project currently provides two main artifacts:
    entries satisfy the constraints placed on their respective tables or not.
    (Note that the CLI is intended for testing and experimentation, not for
    production use.)
+   
+**_Check out [these slides](docs/2020-06-03_p4rt-wg.pdf) for a tour of p4-constraints._**
```

Comment:
Maybe we could add something like "(up to date as of 2020-06-03)"

Reply:
Mh, do we need this given that the date is already on the first slide and also in the file name? Happy to add it if you think it's nonetheless useful, though.

Reply:
I think we should. This will get stale quickly, and we don't want people to think the language is much smaller than it really is if we extend it :)

Reply:
Done